branchpro/apps/_inference.py

```diff
@@ -0,0 +1,195 @@
+#
+# InferenceApp
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+import pandas as pd
+import dash
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+import dash_html_components as html
+
+import branchpro as bp
+from branchpro.apps import IncidenceNumberSimulationApp
+
+
+class BranchProInferenceApp(IncidenceNumberSimulationApp):
+    """BranchProInferenceApp Class:
+    Class for the inference dash app with figure and sliders for the
+    BranchPro models.
+    """
+    def __init__(self):
+        self.app = dash.Dash(
```

Comment:
```suggestion
        super(BranchProInferenceApp, self).__init__()
        self.plot1 = self.plot
```
This will run the other class init. Thus, you don't have to do 
```
        self.app = dash.Dash(
        self.plot2.figure['layout']['legend']['uirevision'] = True
        self.sliders = bp._SliderComponent()
```
It doesn't seem important for now, but it might be helpful when we try to merge the mathjax and text things.

Reply:
Actually you do need to set the uirevision on plot2